README.rst

```diff
@@ -41,13 +45,13 @@ PyPI <https://pypi.org/project/adafruit-circuitpython-tmp117/>`_. To install for
 
 .. code-block:: shell
 
-    pip3 install adafruit-circuitpython-tmp117
+    pip3 install git+https://github.com/ami3go/Adafruit_CircuitPython_TMP11X.git
```

Comment:
I don't think you mean to do this.
```suggestion
    pip3 install adafruit-circuitpython-tmp11x
```

Reply:
Regarding readme . i will correct it if you going to merge it. 
just for now it would be confusing if someone going to install original lib and it will not work with tmp116